Anime-App/Feature/Anime/View/AnimeDetailViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  AnimeDetailViewController.swift
+//  Anime-App
+//
+//  Created by Tobi Adegoroye on 09/08/2023.
+//
+
+import UIKit
+import Kingfisher
+
+
+class AnimeDetailViewController: UIViewController {
+    private lazy var mainStackview: UIStackView = {
+        let stackview = UIStackView()
+        stackview.axis = .vertical
+        stackview.alignment = .fill
+        stackview.distribution = .fillEqually
+        stackview.translatesAutoresizingMaskIntoConstraints = false
+        stackview.spacing = 8
+        return stackview
+    }()
+    
+    private lazy var image: UIImageView = {
```

Comment:
Always end your view with either View or Vw don't treat them as variables or else there is no way for you to know it's a view or not and it makes it clear